app/services/region_backfill.rb

```diff
@@ -44,8 +44,8 @@ def create_regions
     end
     root_type = find_region_type("Root")
     org_type = find_region_type("Organization")
-    facility_group_type = find_region_type("FacilityGroup")
-    block_type = find_region_type("Block")
```

Comment:
I was thinking about this some more and had some good discussions w/ @claudiovallejo about upcoming block level reports.  

I think that if we are using the term "Block" in the UI for IHCI reports, we should stick with Block as the terminology in our code. Its the word that we will be using most often when discussing with designers or users of the system, so it makes sense to keep the name in our Region hierarchy consistent with that.

We can also watch for [an answer](https://simpledotorg.slack.com/archives/CK95QFJA2/p1603305906080800) from Praveen to help get some more confidence in this name.